Use 4 GPUs!
{'train': 'Num of clips:2284', 'val': 'Num of clips:910'}
Epoch 0/19
----------
train Loss: 3.1094 Acc: 0.1038
val Loss: 2.8889 Acc: 0.2077

Epoch 1/19
----------
train Loss: 2.6167 Acc: 0.3809
val Loss: 2.5084 Acc: 0.3297

Epoch 2/19
----------
train Loss: 2.1629 Acc: 0.5044
val Loss: 2.2295 Acc: 0.3923

Epoch 3/19
----------
train Loss: 1.8034 Acc: 0.6033
val Loss: 1.9869 Acc: 0.4538

Epoch 4/19
----------
train Loss: 1.4704 Acc: 0.6891
val Loss: 1.8211 Acc: 0.4681

Epoch 5/19
----------
train Loss: 1.2215 Acc: 0.7360
val Loss: 1.7298 Acc: 0.4890

Epoch 6/19
----------
train Loss: 1.0122 Acc: 0.7894
val Loss: 1.6358 Acc: 0.5110

Epoch 7/19
----------
train Loss: 0.8315 Acc: 0.8415
val Loss: 1.6125 Acc: 0.5099

Epoch 8/19
----------
train Loss: 0.7090 Acc: 0.8643
val Loss: 1.5797 Acc: 0.5121

Epoch 9/19
----------
train Loss: 0.5825 Acc: 0.9024
val Loss: 1.5331 Acc: 0.5396

Epoch 10/19
----------
train Loss: 0.4659 Acc: 0.9343
val Loss: 1.5071 Acc: 0.5440

Epoch 11/19
----------
train Loss: 0.3920 Acc: 0.9488
val Loss: 1.4763 Acc: 0.5516

Epoch 12/19
----------
train Loss: 0.3369 Acc: 0.9694
val Loss: 1.4905 Acc: 0.5637

Epoch 13/19
----------
train Loss: 0.2633 Acc: 0.9790
val Loss: 1.4335 Acc: 0.5626

Epoch 14/19
----------
train Loss: 0.2102 Acc: 0.9864
val Loss: 1.4587 Acc: 0.5604

Epoch 15/19
----------
train Loss: 0.1810 Acc: 0.9917
val Loss: 1.4279 Acc: 0.5714

Epoch 16/19
----------
train Loss: 0.1613 Acc: 0.9891
val Loss: 1.4096 Acc: 0.5791

Epoch 17/19
----------
train Loss: 0.1286 Acc: 0.9969
val Loss: 1.4299 Acc: 0.5703

Epoch 18/19
----------
train Loss: 0.1232 Acc: 0.9961
val Loss: 1.4151 Acc: 0.5681

Epoch 19/19
----------
train Loss: 0.1069 Acc: 0.9956
val Loss: 1.4574 Acc: 0.5626

Training complete in 20m 35s
Best val Acc: 0.579121
Saved the weights of the best epoch in /home/acb11711tx/lzq/ExpEval/model_param/ucf101_24_r2p1d_16_Full_la_0.05_kar_LongRange.pt
Num of clips:910
 Val: Acc@1 57.912 Acc@5 87.033

Normalized confusion matrix
Traceback (most recent call last):
  File "model_train/retrain_ucf101_24_model.py", line 172, in <module>
    cm_fig.savefig(os.path.join(proj_root, 'visual_res', 'cm', f'{save_label}.png'))
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/figure.py", line 2062, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backend_bases.py", line 2263, in print_figure
    **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backends/backend_agg.py", line 530, in print_png
    with cbook.open_file_cm(filename_or_obj, "wb") as fh:
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 626, in open_file_cm
    fh, opened = to_filehandle(path_or_file, mode, True, encoding)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 611, in to_filehandle
    fh = io.open(fname, flag, encoding=encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/acb11711tx/lzq/ExpEval/visual_res/cm/ucf101_24_r2p1d_16_Full_la_0.05_kar_LongRange.png'

Use 4 GPUs!
{'train': 'Num of clips:2284', 'val': 'Num of clips:910'}
Epoch 0/19
----------
train Loss: 3.0663 Acc: 0.1208
val Loss: 2.8562 Acc: 0.2099

Epoch 1/19
----------
train Loss: 2.5288 Acc: 0.3984
val Loss: 2.4548 Acc: 0.3835

Epoch 2/19
----------
train Loss: 2.0903 Acc: 0.5311
val Loss: 2.1890 Acc: 0.4242

Epoch 3/19
----------
train Loss: 1.7282 Acc: 0.6353
val Loss: 1.9690 Acc: 0.4615

Epoch 4/19
----------
train Loss: 1.4025 Acc: 0.7080
val Loss: 1.8012 Acc: 0.4967

Epoch 5/19
----------
train Loss: 1.1558 Acc: 0.7583
val Loss: 1.7131 Acc: 0.4967

Epoch 6/19
----------
train Loss: 0.9601 Acc: 0.8100
val Loss: 1.6034 Acc: 0.5286

Epoch 7/19
----------
train Loss: 0.7832 Acc: 0.8542
val Loss: 1.5491 Acc: 0.5308

Epoch 8/19
----------
train Loss: 0.6420 Acc: 0.8879
val Loss: 1.5225 Acc: 0.5396

Epoch 9/19
----------
train Loss: 0.5288 Acc: 0.9173
val Loss: 1.4988 Acc: 0.5626

Epoch 10/19
----------
train Loss: 0.4292 Acc: 0.9409
val Loss: 1.4107 Acc: 0.5791

Epoch 11/19
----------
train Loss: 0.3439 Acc: 0.9597
val Loss: 1.4093 Acc: 0.5791

Epoch 12/19
----------
train Loss: 0.2895 Acc: 0.9720
val Loss: 1.3705 Acc: 0.5923

Epoch 13/19
----------
train Loss: 0.2489 Acc: 0.9825
val Loss: 1.3462 Acc: 0.6110

Epoch 14/19
----------
train Loss: 0.2053 Acc: 0.9847
val Loss: 1.3640 Acc: 0.5956

Epoch 15/19
----------
train Loss: 0.1689 Acc: 0.9917
val Loss: 1.3522 Acc: 0.6088

Epoch 16/19
----------
train Loss: 0.1466 Acc: 0.9952
val Loss: 1.3700 Acc: 0.6055

Epoch 17/19
----------
train Loss: 0.1204 Acc: 0.9969
val Loss: 1.3485 Acc: 0.5923

Epoch 18/19
----------
train Loss: 0.1123 Acc: 0.9969
val Loss: 1.3150 Acc: 0.5989

Epoch 19/19
----------
train Loss: 0.0958 Acc: 0.9974
val Loss: 1.3436 Acc: 0.6066

Training complete in 21m 56s
Best val Acc: 0.610989
Saved the weights of the best epoch in /home/acb11711tx/lzq/ExpEval/model_param/ucf101_24_r2p1d_16_Full_la_0.1_kar_LongRange.pt
Num of clips:910
 Val: Acc@1 61.099 Acc@5 88.242

Normalized confusion matrix
Traceback (most recent call last):
  File "model_train/retrain_ucf101_24_model.py", line 172, in <module>
    cm_fig.savefig(os.path.join(proj_root, 'visual_res', 'cm', f'{save_label}.png'))
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/figure.py", line 2062, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backend_bases.py", line 2263, in print_figure
    **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backends/backend_agg.py", line 530, in print_png
    with cbook.open_file_cm(filename_or_obj, "wb") as fh:
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 626, in open_file_cm
    fh, opened = to_filehandle(path_or_file, mode, True, encoding)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 611, in to_filehandle
    fh = io.open(fname, flag, encoding=encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/acb11711tx/lzq/ExpEval/visual_res/cm/ucf101_24_r2p1d_16_Full_la_0.1_kar_LongRange.png'
Use 4 GPUs!
{'train': 'Num of clips:2284', 'val': 'Num of clips:910'}
Epoch 0/19
----------
train Loss: 3.0430 Acc: 0.1335
val Loss: 2.7739 Acc: 0.2945

Epoch 1/19
----------
train Loss: 2.4620 Acc: 0.4343
val Loss: 2.3063 Acc: 0.4231

Epoch 2/19
----------
train Loss: 1.9431 Acc: 0.5985
val Loss: 1.9472 Acc: 0.5121

Epoch 3/19
----------
train Loss: 1.5204 Acc: 0.6961
val Loss: 1.7008 Acc: 0.5462

Epoch 4/19
----------
train Loss: 1.2064 Acc: 0.7631
val Loss: 1.5305 Acc: 0.5791

Epoch 5/19
----------
train Loss: 0.9370 Acc: 0.8402
val Loss: 1.3967 Acc: 0.6011

Epoch 6/19
----------
train Loss: 0.7492 Acc: 0.8748
val Loss: 1.3020 Acc: 0.6363

Epoch 7/19
----------
train Loss: 0.6020 Acc: 0.8997
val Loss: 1.2291 Acc: 0.6440

Epoch 8/19
----------
train Loss: 0.4913 Acc: 0.9273
val Loss: 1.1655 Acc: 0.6527

Epoch 9/19
----------
train Loss: 0.3849 Acc: 0.9505
val Loss: 1.1244 Acc: 0.6648

Epoch 10/19
----------
train Loss: 0.3290 Acc: 0.9619
val Loss: 1.0938 Acc: 0.6758

Epoch 11/19
----------
train Loss: 0.2529 Acc: 0.9746
val Loss: 1.0407 Acc: 0.6857

Epoch 12/19
----------
train Loss: 0.2009 Acc: 0.9856
val Loss: 1.0236 Acc: 0.6934

Epoch 13/19
----------
train Loss: 0.1833 Acc: 0.9908
val Loss: 1.0017 Acc: 0.6978

Epoch 14/19
----------
train Loss: 0.1567 Acc: 0.9904
val Loss: 0.9973 Acc: 0.7077

Epoch 15/19
----------
train Loss: 0.1297 Acc: 0.9921
val Loss: 0.9681 Acc: 0.7099

Epoch 16/19
----------
train Loss: 0.1127 Acc: 0.9961
val Loss: 0.9691 Acc: 0.7088

Epoch 17/19
----------
train Loss: 0.0931 Acc: 0.9982
val Loss: 0.9637 Acc: 0.7143

Epoch 18/19
----------
train Loss: 0.0860 Acc: 0.9969
val Loss: 0.9540 Acc: 0.7176

Epoch 19/19
----------
train Loss: 0.0811 Acc: 0.9978
val Loss: 0.9734 Acc: 0.7165

Training complete in 21m 48s
Best val Acc: 0.717582
Saved the weights of the best epoch in /home/acb11711tx/lzq/ExpEval/model_param/ucf101_24_r2p1d_16_Full_la_0.3_kar_LongRange.pt
Num of clips:910
 Val: Acc@1 71.758 Acc@5 93.626

Normalized confusion matrix
Traceback (most recent call last):
  File "model_train/retrain_ucf101_24_model.py", line 172, in <module>
    cm_fig.savefig(os.path.join(proj_root, 'visual_res', 'cm', f'{save_label}.png'))
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/figure.py", line 2062, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backend_bases.py", line 2263, in print_figure
    **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backends/backend_agg.py", line 530, in print_png
    with cbook.open_file_cm(filename_or_obj, "wb") as fh:
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 626, in open_file_cm
    fh, opened = to_filehandle(path_or_file, mode, True, encoding)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 611, in to_filehandle
    fh = io.open(fname, flag, encoding=encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/acb11711tx/lzq/ExpEval/visual_res/cm/ucf101_24_r2p1d_16_Full_la_0.3_kar_LongRange.png'
Use 4 GPUs!
{'train': 'Num of clips:2284', 'val': 'Num of clips:910'}
Epoch 0/19
----------
train Loss: 3.0250 Acc: 0.1419
val Loss: 2.7420 Acc: 0.2626

Epoch 1/19
----------
train Loss: 2.3543 Acc: 0.4812
val Loss: 2.2015 Acc: 0.4648

Epoch 2/19
----------
train Loss: 1.7702 Acc: 0.6799
val Loss: 1.7573 Acc: 0.5802

Epoch 3/19
----------
train Loss: 1.3236 Acc: 0.7636
val Loss: 1.4639 Acc: 0.6363

Epoch 4/19
----------
train Loss: 0.9947 Acc: 0.8240
val Loss: 1.2564 Acc: 0.6835

Epoch 5/19
----------
train Loss: 0.7530 Acc: 0.8831
val Loss: 1.1111 Acc: 0.7143

Epoch 6/19
----------
train Loss: 0.5797 Acc: 0.9151
val Loss: 0.9963 Acc: 0.7440

Epoch 7/19
----------
train Loss: 0.4635 Acc: 0.9339
val Loss: 0.9215 Acc: 0.7451

Epoch 8/19
----------
train Loss: 0.3639 Acc: 0.9558
val Loss: 0.8558 Acc: 0.7692

Epoch 9/19
----------
train Loss: 0.3042 Acc: 0.9632
val Loss: 0.8311 Acc: 0.7571

Epoch 10/19
----------
train Loss: 0.2376 Acc: 0.9764
val Loss: 0.7819 Acc: 0.7769

Epoch 11/19
----------
train Loss: 0.2007 Acc: 0.9851
val Loss: 0.7562 Acc: 0.7780

Epoch 12/19
----------
train Loss: 0.1744 Acc: 0.9869
val Loss: 0.7142 Acc: 0.7857

Epoch 13/19
----------
train Loss: 0.1439 Acc: 0.9904
val Loss: 0.6905 Acc: 0.7967

Epoch 14/19
----------
train Loss: 0.1284 Acc: 0.9930
val Loss: 0.6801 Acc: 0.8000

Epoch 15/19
----------
train Loss: 0.1179 Acc: 0.9921
val Loss: 0.6736 Acc: 0.7934

Epoch 16/19
----------
train Loss: 0.0969 Acc: 0.9947
val Loss: 0.6658 Acc: 0.7989

Epoch 17/19
----------
train Loss: 0.0812 Acc: 0.9987
val Loss: 0.6418 Acc: 0.8132

Epoch 18/19
----------
train Loss: 0.0719 Acc: 0.9965
val Loss: 0.6443 Acc: 0.8088

Epoch 19/19
----------
train Loss: 0.0647 Acc: 0.9996
val Loss: 0.6291 Acc: 0.8088

Training complete in 21m 32s
Best val Acc: 0.813187
Saved the weights of the best epoch in /home/acb11711tx/lzq/ExpEval/model_param/ucf101_24_r2p1d_16_Full_la_0.5_kar_LongRange.pt
Num of clips:910
 Val: Acc@1 81.319 Acc@5 96.813

Normalized confusion matrix
Traceback (most recent call last):
  File "model_train/retrain_ucf101_24_model.py", line 172, in <module>
    cm_fig.savefig(os.path.join(proj_root, 'visual_res', 'cm', f'{save_label}.png'))
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/figure.py", line 2062, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backend_bases.py", line 2263, in print_figure
    **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backends/backend_agg.py", line 530, in print_png
    with cbook.open_file_cm(filename_or_obj, "wb") as fh:
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 626, in open_file_cm
    fh, opened = to_filehandle(path_or_file, mode, True, encoding)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 611, in to_filehandle
    fh = io.open(fname, flag, encoding=encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/acb11711tx/lzq/ExpEval/visual_res/cm/ucf101_24_r2p1d_16_Full_la_0.5_kar_LongRange.png'
Use 4 GPUs!
{'train': 'Num of clips:2284', 'val': 'Num of clips:910'}
Epoch 0/19
----------
train Loss: 3.0398 Acc: 0.1515
val Loss: 2.7206 Acc: 0.3165

Epoch 1/19
----------
train Loss: 2.3173 Acc: 0.5381
val Loss: 2.1013 Acc: 0.5868

Epoch 2/19
----------
train Loss: 1.7064 Acc: 0.6913
val Loss: 1.6275 Acc: 0.6495

Epoch 3/19
----------
train Loss: 1.2345 Acc: 0.7999
val Loss: 1.3036 Acc: 0.7209

Epoch 4/19
----------
train Loss: 0.8989 Acc: 0.8722
val Loss: 1.0420 Acc: 0.7462

Epoch 5/19
----------
train Loss: 0.6757 Acc: 0.8962
val Loss: 0.9007 Acc: 0.7747

Epoch 6/19
----------
train Loss: 0.5023 Acc: 0.9356
val Loss: 0.7833 Acc: 0.8011

Epoch 7/19
----------
train Loss: 0.3988 Acc: 0.9518
val Loss: 0.6981 Acc: 0.8110

Epoch 8/19
----------
train Loss: 0.3139 Acc: 0.9641
val Loss: 0.6443 Acc: 0.8264

Epoch 9/19
----------
train Loss: 0.2509 Acc: 0.9799
val Loss: 0.6020 Acc: 0.8275

Epoch 10/19
----------
train Loss: 0.1943 Acc: 0.9873
val Loss: 0.5642 Acc: 0.8549

Epoch 11/19
----------
train Loss: 0.1653 Acc: 0.9869
val Loss: 0.5349 Acc: 0.8615

Epoch 12/19
----------
train Loss: 0.1434 Acc: 0.9904
val Loss: 0.5048 Acc: 0.8484

Epoch 13/19
----------
train Loss: 0.1180 Acc: 0.9943
val Loss: 0.4924 Acc: 0.8516

Epoch 14/19
----------
train Loss: 0.1004 Acc: 0.9952
val Loss: 0.4800 Acc: 0.8560

Epoch 15/19
----------
train Loss: 0.0960 Acc: 0.9947
val Loss: 0.4652 Acc: 0.8571

Epoch 16/19
----------
train Loss: 0.0794 Acc: 0.9978
val Loss: 0.4562 Acc: 0.8681

Epoch 17/19
----------
train Loss: 0.0771 Acc: 0.9969
val Loss: 0.4582 Acc: 0.8791

Epoch 18/19
----------
train Loss: 0.0693 Acc: 0.9969
val Loss: 0.4361 Acc: 0.8758

Epoch 19/19
----------
train Loss: 0.0639 Acc: 0.9974
val Loss: 0.4416 Acc: 0.8769

Training complete in 21m 30s
Best val Acc: 0.879121
Saved the weights of the best epoch in /home/acb11711tx/lzq/ExpEval/model_param/ucf101_24_r2p1d_16_Full_la_0.7_kar_LongRange.pt
Num of clips:910
 Val: Acc@1 87.912 Acc@5 97.802

Normalized confusion matrix
Traceback (most recent call last):
  File "model_train/retrain_ucf101_24_model.py", line 172, in <module>
    cm_fig.savefig(os.path.join(proj_root, 'visual_res', 'cm', f'{save_label}.png'))
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/figure.py", line 2062, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backend_bases.py", line 2263, in print_figure
    **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backends/backend_agg.py", line 530, in print_png
    with cbook.open_file_cm(filename_or_obj, "wb") as fh:
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 626, in open_file_cm
    fh, opened = to_filehandle(path_or_file, mode, True, encoding)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 611, in to_filehandle
    fh = io.open(fname, flag, encoding=encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/acb11711tx/lzq/ExpEval/visual_res/cm/ucf101_24_r2p1d_16_Full_la_0.7_kar_LongRange.png'
Use 4 GPUs!
{'train': 'Num of clips:2284', 'val': 'Num of clips:910'}
Epoch 0/19
----------
train Loss: 2.9467 Acc: 0.2123
val Loss: 2.5004 Acc: 0.4582

Epoch 1/19
----------
train Loss: 2.0848 Acc: 0.6616
val Loss: 1.6612 Acc: 0.7407

Epoch 2/19
----------
train Loss: 1.3063 Acc: 0.8494
val Loss: 1.0313 Acc: 0.8440

Epoch 3/19
----------
train Loss: 0.8004 Acc: 0.9212
val Loss: 0.7105 Acc: 0.8846

Epoch 4/19
----------
train Loss: 0.5223 Acc: 0.9553
val Loss: 0.5571 Acc: 0.8824

Epoch 5/19
----------
train Loss: 0.3714 Acc: 0.9628
val Loss: 0.4713 Acc: 0.8945

Epoch 6/19
----------
train Loss: 0.2780 Acc: 0.9750
val Loss: 0.4012 Acc: 0.8978

Epoch 7/19
----------
train Loss: 0.2203 Acc: 0.9812
val Loss: 0.3631 Acc: 0.9055

Epoch 8/19
----------
train Loss: 0.1687 Acc: 0.9851
val Loss: 0.3460 Acc: 0.9099

Epoch 9/19
----------
train Loss: 0.1584 Acc: 0.9904
val Loss: 0.3292 Acc: 0.9132

Epoch 10/19
----------
train Loss: 0.1202 Acc: 0.9926
val Loss: 0.3119 Acc: 0.9187

Epoch 11/19
----------
train Loss: 0.0991 Acc: 0.9930
val Loss: 0.3081 Acc: 0.9099

Epoch 12/19
----------
train Loss: 0.1023 Acc: 0.9939
val Loss: 0.2800 Acc: 0.9165

Epoch 13/19
----------
train Loss: 0.0764 Acc: 0.9961
val Loss: 0.2744 Acc: 0.9187

Epoch 14/19
----------
train Loss: 0.0787 Acc: 0.9965
val Loss: 0.2749 Acc: 0.9231

Epoch 15/19
----------
train Loss: 0.0672 Acc: 0.9978
val Loss: 0.2603 Acc: 0.9264

Epoch 16/19
----------
train Loss: 0.0582 Acc: 0.9982
val Loss: 0.2676 Acc: 0.9231

Epoch 17/19
----------
train Loss: 0.0597 Acc: 0.9969
val Loss: 0.2664 Acc: 0.9231

Epoch 18/19
----------
train Loss: 0.0528 Acc: 0.9974
val Loss: 0.2565 Acc: 0.9198

Epoch 19/19
----------
train Loss: 0.0461 Acc: 0.9982
val Loss: 0.2464 Acc: 0.9275

Training complete in 21m 10s
Best val Acc: 0.927473
Saved the weights of the best epoch in /home/acb11711tx/lzq/ExpEval/model_param/ucf101_24_r2p1d_16_Full_la_0.9_kar_LongRange.pt
Num of clips:910
 Val: Acc@1 92.747 Acc@5 99.451

Normalized confusion matrix
Traceback (most recent call last):
  File "model_train/retrain_ucf101_24_model.py", line 172, in <module>
    cm_fig.savefig(os.path.join(proj_root, 'visual_res', 'cm', f'{save_label}.png'))
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/figure.py", line 2062, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backend_bases.py", line 2263, in print_figure
    **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backends/backend_agg.py", line 530, in print_png
    with cbook.open_file_cm(filename_or_obj, "wb") as fh:
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 626, in open_file_cm
    fh, opened = to_filehandle(path_or_file, mode, True, encoding)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 611, in to_filehandle
    fh = io.open(fname, flag, encoding=encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/acb11711tx/lzq/ExpEval/visual_res/cm/ucf101_24_r2p1d_16_Full_la_0.9_kar_LongRange.png'
